### ISSUE TITLE
System.InvalidOperationException: Validation of K5:K2551 failed: Formula2 must be set if operator is 'between' or 'notBetween' 

### ISSUE THREAD

**Issue by vijam** [EPPlusSoftware/EPPlus#1166]
```
 public static void AddEmailFormatValidation(
            this ExcelWorksheet worksheet,
            ExcelRange range,
            string errorTitle,
            decimal min,
            decimal max)
        {
            var validation = worksheet.DataValidations.AddCustomValidation(range.Address);
            validation.ShowErrorMessage = true;
            validation.ErrorTitle = errorTitle;
            validation.ErrorStyle = ExcelDataValidationWarningStyle.stop;
            validation.Error = $"Invalid Email Format or Value must be in the {min} to {max} range.";
            validation.Formula.ExcelFormula = $"AND(ISNUMBER(SEARCH(\"@\", {range.Address})), ISNUMBER(SEARCH(\".\", {range.Address})), LEN({range.Address}) > 0)";
            validation.AllowBlank = true;
        }
```
Above AddCusotmvaliadtion is  exposed only only one for ExcelFormula , but while saving the excel caught in exception as System.InvalidOperationException: Validation of K5:K2551 failed: Formula2 must be set if operator is 'between' or 'notBetween' 

version used 4.5.31 is any fix availble in higher vesions  or please suggest the fix

**Comment by vijam** [comment]
validate in excel for custom it requires only one formula 
![image](https://github.com/EPPlusSoftware/EPPlus/assets/10205227/8c815be0-39d8-4759-bcb3-adf3dc008cd5)

**Comment by OssianEPPlus** [comment]
We do not support version 4.5 as it is deprecated as of 4 years ago.
We made significant improvements to DataValidations in epplus6. I've verified and this bug is fixed in both epplus 6.2.12 and the recently released epplus 7.0.1
